packages/core/src/cart/cart-request-sender.ts

```diff
@@ -19,4 +21,41 @@ export default class CartRequestSender {
 
         return this._requestSender.post(url, { body, headers, timeout });
     }
+
+    async loadCart(
+        cartId: string,
+        host?: string,
+        options?: RequestOptions,
+    ): Promise<Response<Cart | undefined>> {
+        const path = 'cart-information';
+        const url = host ? `${host}/${path}` : `/${path}`;
+
+        const requestOptions: RequestOptions = {
+            ...options,
+            params: {
+                cartId,
+            },
+        };
+
+        return this._requestSender
+            .get<HeadlessCartRequestResponse>(url, {
+                ...requestOptions,
+            })
+            .then(this.transformToCartResponse);
```

Comment:
Lets keep one style for handling promises if you are using async

```suggestion
        const response = await this._requestSender
            .get<HeadlessCartRequestResponse>(url, {
                ...requestOptions,
            });
        
        return this.transformToCartResponse(response);    
```